.gitignore

```diff
@@ -45,6 +45,8 @@ hphp/hack/cargo_home/
 /hphp/ffi/java/classes
```

Comment:
I had to make all sorts of updates to my `.gitignore`, and I'm not totally sure why. Maybe something about the way I'm building HHVM is different from how y'all do it?